content/influxdb/v2.0/reference/cli/influx/v1/auth/create.md

```diff
@@ -30,6 +30,7 @@ influx v1 auth create [flags]
 |      | `--no-password`   | Don't prompt for a password. (You must use the `v1 auth set-password` command before using the token.) |             |                         |
 | `-o` | `--org`           | Organization name                                                                                      | string      | `$INFLUX_ORG`           |
 |      | `--org-id`        | Organization ID                                                                                        | string      | `$INFLUX_ORG_ID`        |
+|      | `--password`      | Password to set on this token                                                                          |             |                         |
```

Comment:
Should "token" be "authorization?

Reply:
I think it should. I think what the v1 auth system does (and I may be totally wrong) is create a user/password combo that can be used for basic authentication in the compatibility API, which acts as an alias for a token. But the token itself doesn't have a password. I know `token` is what's in the source, but I don't want to users to get confuse v1 auths and tokens.

Reply:
Thanks. This sounds right to me.